.github/workflows/ci.yml

```diff
@@ -39,6 +41,60 @@ jobs:
         run: make VERBOSE=1 check
       - name: distcheck
         run: make VERBOSE=1 distcheck
+      - name: distribution tarball is complete
+        run: |
+          set -xe
+          git_ver=$(dpkg-query -f '${Version}' -W git) || exit 1
+          # actions/checkout@v2 only makes a clone if Git is v2.18 or
+          # later, and this test requires a clone.
+          dpkg --compare-versions "${git_ver}" ge '1:2.18~' || {
+              printf '::warning::%s\n' "This test requires Git v2.18 or later"
+              exit 0
+          }
+          tmpdir=$(mktemp -d) || exit 1
+          mv ddclient-*.tar.gz "${tmpdir}"
+          git archive -o "${tmpdir}"/git-repo.tar.gz HEAD
+          mkdir "${tmpdir}"/git-repo
+          cd "${tmpdir}"/git-repo
+          tar xvfa ../git-repo.tar.gz
+          # Delete files checked into Git that shouldn't be in the
+          # distribution tarball.
+          rm -rf \
+              .github \
+              .gitignore \
+              docs/ipv6-design-doc.md \
```

Comment:
```suggestion
              docs/ \
```

Reply:
We might add docs here that we do want included in the tarball.